chat-server/seed-content/index.ts

```diff
@@ -4,9 +4,7 @@ import { Content } from "../src/services/content";
 import { RecursiveCharacterTextSplitter } from "langchain/text_splitter";
 import { Document } from "langchain/dist/document";
 import GPT3Tokenizer from "gpt3-tokenizer";
-import { embeddings } from "../src/services/embeddings";
-import { ObjectId } from "mongodb";
-import { mongodb } from "../src/integrations/mongodb";
+import { ObjectId, embeddings, mongodb } from "chat-core";
```

Comment:
as we discussed before, we should refactor these to not be exported singletons..but subject of a future PR there